include/devicetree/gpio.h

```diff
@@ -302,4 +329,4 @@ extern "C" {
 }
 #endif
 
-#endif  /* ZEPHYR_INCLUDE_DEVICETREE_GPIO_H_ */
+#endif /* ZEPHYR_INCLUDE_DEVICETREE_GPIO_H_ */
```

Comment:
I am not really sure why all these seemingly unrelated gpio.h changes are in this PR? Was it a mistake or a leftover?

Reply:
I used them before but switched to the _dt variants last minute. I will remove them.